docs/cmake-project-view.md

```diff
@@ -0,0 +1,116 @@
+# CMake Project Outline View
+
+Starting in CMake Tools 1.16, we will provide users the ability to customize their status bar with related CMake items like presets, kits, variants, build, configure, and launch to allow for users to de-clutter this global status bar as much as they would like to, while also having quick access to commonly performed CMake actions.
+
+Users can specify what is shown in the status bar through their `settings.json` file.
+
+The default settings will be set to the following:
+## Default Settings Json
+```json
+"cmake.status.statusBarVisibility": "hidden"
+
+"cmake.status.advanced": {
```

Comment:
technically there aren't spaces between the settings but it doesn't really matter (nit)